cluster/gce/gci/configure-helper.sh

```diff
@@ -724,15 +714,12 @@ function create-master-pki {
   fi
 }
 
-# After the first boot and on upgrade, these files exist on the master-pd
-# and should never be touched again (except perhaps an additional service
-# account, see NB below.) One exception is if METADATA_CLOBBERS_CONFIG is
-# enabled.
+# Master auth files are stateless, and are created on every master VM bootstrap.
 function create-master-auth {
   echo "Creating master auth files"
   local -r auth_dir="/etc/srv/kubernetes"
   local -r known_tokens_csv="${auth_dir}/known_tokens.csv"
-  if [[ -e "${known_tokens_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
```

Comment:
It would be helpful to know why this was added in a1209067562b328d85c265e752a1f78b045c6cfb

cc @mikedanese

Reply:
From the original issue https://github.com/kubernetes/kubernetes/issues/49895 it seems like persisting the `known_tokens.csv` file disk was a problem. By removing it from PD, master reboot can mitigate the original issue.